### DISCUSSION TITLE
DevOps

### DISCUSSION THREAD

**Discussion by inventors** [saleor/saleor#9942] [General]
Good afternoon,

I am looking for experienced Saleor DevOps to update my Saleor application from time to time. 

Must have experience with both the front & back end technologies, ability to fix issues and AWS solution architect will be good too. 

Anyone interested please contact me.

Thanks

**Comment by timuric** [comment]
@inventors you could check out Saleor cloud which cost-effective way to automate dev-ops and more. 
One limitation is that you can't run a forked version of Saleor, however, our recommended way to extend Saleor is by using Apps instead of forking which makes it easier to stay up to date with the latest features.
